src/Microsoft.OpenApi/Validations/Rules/OpenApiPathsRules.cs

```diff
@@ -42,15 +42,15 @@ public static class OpenApiPathsRules
             new ValidationRule<OpenApiPaths>(nameof(PathMustBeUnique),
                 (context, item) =>
                 {
-                    var hashSet = new HashSet<string>();
+                    var HashSet = new HashSet<string>();
```

Comment:
I'm not sure the casing change brings any value here and it collides with the call name, revert to avoid confusion?

Reply:
Yeah my bad this was an issue with using replace all